client/admin/import/ImportHistoryPage.js

```diff
@@ -47,10 +47,10 @@ function ImportHistoryPage() {
 		};
 
 		loadData();
-	}, []);
+	}, [dispatchToastMessage, getCurrentImportOperation, getLatestImportOperations, setCurrentOperation, setLatestOperations, setLoading, t]);
 
 	const hasAnySuccessfulSlackImport = useMemo(() =>
-		latestOperations?.some(({ importerKey, status }) => importerKey === 'slack' && status === ProgressStep.DONE), [latestOperations]);
+		latestOperations?.some(({ importerKey, status }) => importerKey === 'slack' && status === ProgressStep.DONE), [latestOperations?.some]);
```

Comment:
```suggestion
		latestOperations?.some(({ importerKey, status }) => importerKey === 'slack' && status === ProgressStep.DONE), [latestOperations]);
```

Reply:
Looks like a bug: https://github.com/facebook/react/pull/19062
Let's keep this warning, as it is not critical.